ElectronClient/app/gui/NotebookPropertiesDialog.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+
+const { _ } = require('lib/locale.js');
+const { themeStyle } = require('../theme.js');
+const DialogButtonRow = require('./DialogButtonRow.min');
+
+const Folder = require('lib/models/Folder.js');
+
+interface NotebookPropertiesDialogProps {
+	theme: number,
+	folderId: string,
+	onClose: Function,
+}
+
+export default function NotebookPropertiesDialog(props: NotebookPropertiesDialogProps) {
+	console.info('Render NotebookPropertiesDialog');
+
+	const [folder, formFolder] = useState<any>({});
```

Comment:
Not sure if it's a typo but it should be `const [formFolder, setFormFolder]`

Reply:
Not a typo, just a misunderstanding. form is usually a verb, so I figured you were requesting set, another verb, to be changed to that. It's not quite clear elsewhere in the code that form is being used as an adjective here. I can definitely fix that.

Reply:
Sorry I didn't mean as a verb. "form" in this context means the form where you enter data. So this `formFolder` will have the current state of the form.